packages/database/src/migrations/20210126013657-AddK13C580YPositiveMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+import { insertObject } from '../utilities';
+import { generateId } from '../utilities/generateId';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const MAIN_OVERLAY_ID = 'PG_STRIVE_K13_C580Y_Positive';
+const LINKED_MEASURE_OVERLAY_ID = 'PG_STRIVE_K13_C580Y_Positive_Count';
+const NEW_OVERLAY_GROUP_ID = generateId();
+const NEW_OVERLAY_GROUP_RELATION_FOR_OVERLAY_ID = generateId();
+const NEW_OVERLAY_GROUP_RELATION_FOR_GROUP_ID = generateId();
+
+const BASE_OVERLAY = {
+  userGroup: 'STRIVE User',
+  dataElementCode: 'value',
+  isDataRegional: true,
+  measureBuilderConfig: {
+    dataValues: {
+      STR_K13C05: 1,
+    },
+    programCode: 'SK13C',
+    entityAggregation: {
+      aggregationType: 'REPLACE_ORG_UNIT_WITH_ORG_GROUP',
+      dataSourceEntityType: 'village',
+      aggregationEntityType: 'facility',
+    },
+  },
+  measureBuilder: 'countEventsPerOrgUnit',
+  countryCodes: '{PG}',
+  projectCodes: '{strive,explore}',
+};
+
+const LINKED_MEASURE_OVERLAY = {
+  ...BASE_OVERLAY,
+  id: `${LINKED_MEASURE_OVERLAY_ID}`,
+  name: 'K13 C580Y Positive Count',
+  presentationOptions: {
+    values: [
+      {
+        value: 'other',
+        hideFromLegend: true,
+        color: 'green',
+      },
+      {
+        value: '0',
+        hideFromLegend: true,
+        color: 'red',
+      },
+      {
+        value: null,
```

Comment:
What happens when you click on a grey pin?

Reply:
It just displays "No Data"

Reply:
Awesome, perfect